workflow/resources/igv-report-template.html

```diff
@@ -1,925 +1,864 @@
-		}
-		for (column of sample_columns) {
-		    data.push({data: column})
-		}
-
-                $('#variant_table').DataTable({
-                    data: tableJson,
-                    columns: data,
-                    ordering: true,
-                });
+        }
+
+        function initDataTable() {
+            var samples = Object.keys(tableJson[0]).filter(function (key) { return key.endsWith(":AF") }).map(function (entry) { return entry.slice(0, -3) })
+            var columns = ['Chrom', 'Position', 'ID', 'Cosmic IDs', 'Gene', 'dgiDB drugs', 'DNA alteration', 'Impact']
+            var sample_columns = []
+            for (sample of samples) {
+                sample_columns.push(`${sample}:AF`)
+                sample_columns.push(`${sample}:DP`)
             }
-
-            function parse_dgidb_data(data) {
-                var parsed_drugs = []
-                var entries = data.split(",").map(row => row.split("|")[1])
-                entries = entries.filter(function(entry) {return entry != '.'})
-                const unique_entries = [...new Set(entries)]
-                return unique_entries.join(', ')
+            columns.push(...sample_columns)
+            var table = document.createElement("table");
+            table.id = "variant_table"
+            document.getElementById("tableSelectorDiv").appendChild(table);
+
+            var thead = document.createElement('thead');
+            table.appendChild(thead);
+
+            var headerRow = thead.insertRow(0);
+            for (column of columns) {
+                var cell = document.createElement("th");
+                headerRow.appendChild(cell);
+                cell.innerHTML = column;
             }
-
-            function parse_column_data(data) {
-                var parsed_ids = []
-                var entries = data.split(/[,\|]/)
-                entries = entries.filter(function(entry) {return entry != '.'})
-                const unique_entries = [...new Set(entries)]
-                return unique_entries.join(', ')
+            var data = []
+            for (column of ['CHROM', 'POSITION']) {
+                data.push({ data: column })
             }
-
-            function initDetailsTables() {
-                var table_ids = {
-                    "common_details_table": "tableCommonDetailsDiv",
-                    "variant_details_table": "tableVariantsDetailsDiv"
+            for (column of ['ID', 'cosmic_LEGACY_ID', 'GENE', 'dgiDB_drugs', 'DNA ALTERATION', 'IMPACT']) {
+                if (column == 'dgiDB_drugs') {
+                    data.push({ data: 'dgiDB_drugs', render: $.fn.dataTable.render.multi([parse_dgidb_data, $.fn.dataTable.render.ellipsis(15, true)]) })
+                } else {
+                    data.push({ data: column, render: $.fn.dataTable.render.multi([parse_column_data, $.fn.dataTable.render.ellipsis(15, true)]) })
```

Comment:
Magic happens here.

The rest has been changed by the editor as there was a mix of space and tab identations.